src/antelope/stores/chain.ts

```diff
@@ -220,6 +278,16 @@ export const useChainStore = defineStore(store_name, {
                 throw new AntelopeError('antelope.chain.error_invalid_network', { network });
             }
         },
+        setStakedRatio(label: string, ratio: ethers.BigNumber) {
+            const ratioNumber = parseFloat(ethers.utils.formatUnits(ratio, 18));
```

Comment:
instead of `18` here, we should use systemtoken.decimals, right? or at least `WEI_PRECISION`? same comment for line 287